org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/stratego/StrategoCommon.java

```diff
@@ -112,6 +155,35 @@
             throw handleException(e, runtime, strategy);
         }
     }
+    
+	@Override
+	public IStrategoTerm invoke(HybridInterpreter runtime, IStrategoTerm input, String strategy,
+			List<IStrategoTerm> termArguments) throws MetaborgException {
+		runtime.setCurrent(input);
+		
+		final IStrategoString strategyName = createStrategyName(strategy, termArguments);
+		final IStrategoAppl strategyNameTerm = termFactory.makeAppl("SVar", strategyName);
+		final IStrategoAppl strategyCallTerm = termFactory.makeAppl("CallT", strategyNameTerm, termFactory.makeList(), termFactory.makeList(termArguments));
+		
+        final IStrategoConstructor sdefT = termFactory.makeConstructor("SDefT", 4);
+		final IStrategoAppl strategyTerm = termFactory.makeAppl(sdefT, strategyName, 
+				termFactory.makeList(), termFactory.makeList(), strategyCallTerm);
+		try {
+			if(runtime.evaluate(strategyTerm)) {
+				return runtime.current();
+			}
+		} catch (InterpreterException e) {
+            throw handleException(e, runtime, strategy);
+		}
+		return null;
+	}
```

Comment:
Please use consistent indentation, there is a mix of tabs and spaces in here

Reply:
Done